src/components/Countdown/Countdown.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story } from "@storybook/react"
+import { addLeadingZero } from "../../utils"
+import { H4 } from "../Typography"
+import { Countdown as CountdownComponent, CountdownProps } from "./index"
+
+const targetUnixTimestamp = Math.floor(Date.now() / 1000) + 600
+
+const Template: Story<CountdownProps> = (args) => {
+  return (
+    <CountdownComponent {...args}>
+      {(days, hours, minutes, seconds) => {
```

Comment:
I think it would be nice to show the onComplete functionality. But not mandatory, just a preference

Reply:
Change is not needed anymore since the file doesn't exist. Please see https://github.com/threshold-network/components/pull/12#discussion_r953857549